### PR TITLE
Rob fast forward

### PR DESCRIPTION
**Pull request by RobStafford**
This largely completes the non-interactive parts of my widget (with the exception of 1. correctly resizing various bars, 2. sorting, 3. filtering, 4. showing more reviews, 5. submitting a new review--which I realize will still be a lot of work). ...but it looks good in full screen.

### CODE REVIEW

File: client/src/components/Ratings/components/ProductBreakdown.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ProductBreakdownItem from './ProductBreakdownItem.jsx'
```
Comment (DominicMonares) [inline client/src/components/Ratings/components/ProductBreakdown.jsx]: semi-colon needed

File: client/src/components/Ratings/components/ProductBreakdown.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    return(
+      <div>
+        <ul className="product_breakdown">{product_breakdowns}</ul>
+      </div>
+    )
+
+}
+
+export default ProductBreakdowns;
```
Comment (DominicMonares) [inline client/src/components/Ratings/components/ProductBreakdown.jsx]: newline needed

File: client/src/components/Ratings/components/StarCountBar.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+  return (
+    <li>
+      <br></br>{star_number} stars <span className="star_bar" style={{width: parseInt(star_count)}}></span>
+    </li>
+  );
+
+}
+
+export default StarCount;
```
Comment (DominicMonares) [inline client/src/components/Ratings/components/StarCountBar.jsx]: newline needed

**Review by DominicMonares — CHANGES_REQUESTED** [review]
Looks great! I only see 3 minor changes to be made (1 missing semi-colon and 2 missing newlines). Approved otherwise.

**Comment by RobStafford** [comment]
Requested changes complete.

File: client/src/components/Ratings/components/IndividualReview.jsx [hunk 4]
```diff
@@ -1,34 +1,61 @@
 import React from 'react';
 import moment from 'moment';
 import StarRating from './StarRating.jsx';
+import ReviewImages from './ReviewImages.jsx'
```
Comment (DominicMonares) [inline client/src/components/Ratings/components/IndividualReview.jsx]: Missing semi-colon

**Review by DominicMonares — APPROVED** [review]

**Pull request merged**